public/modules/clients/controllers/clients.client.controller.js

```diff
@@ -2,58 +2,59 @@
 
 // Projects controller
 angular.module('clients').controller('ClientsController', ['$scope', '$stateParams', '$location', 'Authentication', 'Projects', '$upload', 'ngAudio', 'ngAudioGlobals', '$http', '$uibModal', '$rootScope', 'Socket', '$cookies', '$window', 'IS_NOT_MOBILE',
-	function($scope, $stateParams, $location, Authentication, Projects, $upload, ngAudio, ngAudioGlobals, $http, $uibModal, $rootScope, Socket, $cookies, $window, IS_NOT_MOBILE ) {
-
-	$scope.authentication = Authentication;
-
-    // controller vars
-    $scope.projectsTotalCnt = 0;
-	$scope.project = {};
-    $scope.audio = '';
-	$scope.lastAudioID = 0;
-	$scope.audioStatus = 0;
-    $scope.newLead = {};
-    ngAudioGlobals.unlock = false;
-    $scope.clientNotes = '';
-	$scope.projAuditions = [];
-	$scope.newProject = {
-		title: '',
-		estimatedCompletionDate: '',
-		notifyClient: true,
-		client: [],
-		scripts: [],
-		copiedScripts: [],
-		referenceFiles: [],
-		copiedReferenceFiles: [],
-		talent: []
-	};
-    // filter vars
-    $scope.predicate = '';
-    // rating
-	$scope.hide = 0;
-	$scope.max = 5;
-	$scope.isReadonly = false;
-	$scope.ratings = [];
-	$scope.ratingsAvg = [];
-	// projects client portal
-	$scope.selectedAuditions = [];
-	$scope.hideList = [];
-
-	// clear mem leaks on controller destroy
-	$scope.$on('$destroy', function (event) {
-	Socket.removeAllListeners();
-    });
-        
-    $scope.tinymceOptions = {
-        plugins: 'link image code',
-        toolbar: 'undo redo | bold italic | alignleft aligncenter alignright | code'
-    };
-
-	// gathers to field addresses for emails
-	$scope.gatherToAddresses = function(type){
-
-		// create mail object
-		var emailObj = {
+	function ($scope, $stateParams, $location, Authentication, Projects, $upload, ngAudio, ngAudioGlobals, $http, $uibModal, $rootScope, Socket, $cookies, $window, IS_NOT_MOBILE) {
+
+		$scope.authentication = Authentication;
+
+		// controller vars
+		$scope.projectsTotalCnt = 0;
+		$scope.project = {};
+		$scope.audio = '';
+		$scope.lastAudioID = 0;
+		$scope.audioStatus = 0;
+		$scope.newLead = {};
+		ngAudioGlobals.unlock = false;
+		$scope.clientNotes = '';
+		$scope.projAuditions = [];
+		$scope.newProject = {
+			title: '',
+			estimatedCompletionDate: '',
+			notifyClient: true,
+			client: [],
+			scripts: [],
+			copiedScripts: [],
+			referenceFiles: [],
+			copiedReferenceFiles: [],
+			talent: []
+		};
+		// filter vars
+		$scope.predicate = 'file.name';
```

Comment:
These 2 lines are the only changes, i swear i didn't run it through the formatter...